deployment/.template.env

```diff
@@ -18,3 +18,7 @@ REDIS_PASSWORD=redis_password
 
 RABBITMQ_HOST=rabbitmq
 SENTRY_DSN=
+
+SUPPORT_STAFF="dimas@kartoza.com,amy@kartoza.com"
```

Comment:
We shouldn't define this here, maybe add a flag 'is_support_staff' here https://github.com/kartoza/africa_rangeland_watch/blob/main/django_project/base/models.py#L43.

Reply:
@tinashechiraya we can assign user to be 'support' user by adding this flag in the user model.

Reply:
and then you can query like this

```
 support_staff = UserProfile.objects.filter(is_support_staff=True).values_list('email', flat=True)
```
  
Do you see the advantages of doing this instead of environment variables?

Reply:
Yes this is much better